src/services/ExperianService.ts

```diff
@@ -141,14 +141,15 @@ export class ExperianService {
     	} else {
     		try {
     			const endpoint = `${this.experianBaseUrl}${Constants.EXPERIAN_TOKEN_ENDPOINT_PATH}`;
-    			this.logger.info("No valid token found - trying to generate new Experian token", { endpoint });
+    			
     			const params = {
     				username: clientUsername,
     				password: clientPassword,
     				client_id: clientId,
     				client_secret: clientSecret,
     			};
-
+    			this.logger.info("No valid token found - trying to generate new Experian token", { endpoint, params });
```

Comment:
Can we make this 2 commands keep the original with endpoint and new log line for params that is debug